src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -84,6 +84,9 @@ message ConfigProto {
 message LanguageSettingsProto {
   // Package name used for the language.
   string package_name = 1;
+
+  // Location of hand-written client, if any.
+  string hand_written_location = 2;
```

Comment:
I think I would prefer something more along the lines of what I suggested elsewhere: `domain_layer_location`. The wording in the Go docs can be settled separately.